apps/logging/loggers.py

```diff
@@ -1,10 +1,29 @@
 import json
 import logging
 
-from django.conf import settings
-
 from apps.dot_ext.models import get_application_counts, get_application_require_demographic_scopes_count
 from apps.fhir.bluebutton.models import check_crosswalks
+from django.conf import settings
+
+
+"""
+  Logger and logging function for waffle flags, switches
+"""
+waffle_event_logger = logging.getLogger('audit.waffle.event')
+
+
+def log_v2_blocked(user=None, path=None, app=None, err=None, **kwargs):
```

Comment:
I'm not really sure of the filesystem layout yet but does this belong in this file? It looks like this is for global logging, do the app specific loggers belong somewhere else?

Reply:
v2 specific logic are across apps, for this particular waffle flag related logging, there are 2 point of logging: apps/accounts and apps/fhir/bluebutton, so it is OK to put the logging util in common logging?

Reply:
Still not 100% convinced it belongs here. A very specific function for logging out when a v2 endpoint is blocked by a switch. None of our other logging functions are in here except the one. But maybe this is something we can address later. @dtisza1 did you have any thoughts on this?

Reply:
@nbragdon @JFU-GIT I had put the `log_global_state_metrics()` function in that location, because the related management command was located there.

Since this logging is being used under multiple Django apps locations, it does seem like it should live in one of those areas?  Maybe `apps/fhir/bluebutton/loggers.py` ?